detector/wordpress.go

```diff
@@ -196,27 +217,72 @@ func extractToVulnInfos(pkgName string, cves []WpCveInfo) (vinfos []models.VulnI
 				Link: url,
 			})
 		}
+		for _, id := range vulnerability.References.YouTube {
+			refs = append(refs, models.Reference{
+				Link: fmt.Sprintf("https://www.youtube.com/watch?v=%s", id),
+			})
+		}
+
+		var exploits []models.Exploit
+		for _, id := range vulnerability.References.ExploitDB {
+			exploits = append(exploits, models.Exploit{
+				ExploitType: exploitmodels.OffensiveSecurityType,
```

Comment:
I imported exploitmodels referring to the following implementation.

https://github.com/future-architect/vuls/blob/7d5a47bc3354da4d8cfbef2200c916663d72a2b4/models/vulninfos.go#L827-L829

Reply:
It is true that the ExploitType of exploitdb data should be OffensiveSecurity, but this actually functions as a source, so `wpscan` may be preferable at present.
https://github.com/future-architect/vuls/blob/7d5a47bc3354da4d8cfbef2200c916663d72a2b4/models/utils.go#L80-L87

Reply:
Thank you ! You have wonderful expertise, it's very educational.
I have switched from OffensiveSecurity to wpscan.